### DISCUSSION TITLE
Can't login or ssh to Security Onion using OS account created using install

### DISCUSSION THREAD

**Discussion by vanh20** [Security-Onion-Solutions/securityonion#4150] [Unsupported Versions] [answered]
Hello There All! :)

My wife was found to have CA and we had to go through all the testings and way upto her first chemo. It was more than a month since I last logged in and I'm not so sure if I forgot the password of the OS account I created during installation. I can still login on the SO Console via the web interface, but can't ssh nor login directly to SO itself.

Is there a way to reset that user account password? I tried getting inside and changing it via this link "https://www.vultr.com/docs/reset-the-root-password-on-centos". But after rebooting it still won't allow me in using the newly set password. Should I just delete everything and re-install?

TIA,

**Comment by darth-drew** [comment]
Sorry for the news.. I’d follow this guide and be sure to passwd the account you created, which may not have been username root. https://www.unixmen.com/reset-root-password-centos-7/

**Reply by vanh20** [reply]
Hello darth-drew!

   Thanks for the reply. I did exactly as stated on your link. However, for some reason, it created a new problem, **[!!!!!!] Failed to load SELinux policy, freezing**.

I searched for and found this "https://www.virtualramblings.com/failed-to-load-selinux-policy-freezing/". Then, I can now boot and login to the passwd changed account I made.

 However, I have this one problem, If I reboot, I'm back to the old **[!!!!!!] Failed to load SELinux policy, freezing** problem again. Any ideas? Is it safe to make the changes proposed on the link to correct the freeze?

Will also try other solutions out there and will update this once found.



Thanks!

**Comment by vanh20** [comment]
Hello There,

   As suggested here "https://asafshoval.wordpress.com/2014/11/18/overcome-fail-to-load-selinux-policy-freezing-error-message-while-booting-linux/" and here "https://www.thegeekdiary.com/centos-redhat-how-to-set-selinux-modes/" I did change the /etc/sysconfig/selinux file value to **SELINUX=permissive**. I then rebooted, and the system rebooted once more by itself, and I was able to log-in normally to my account.

   Is this the right thing to do? Will I be opening up my installation to some external forces by doing this? I need to hear the experts please.

Thanks everyone!

**Reply by vanh20** [reply]
Hello There once more! :)

   I tried to find out more about /etc/sysconfig/selinux file and arrived at this "https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/4/html/reference_guide/s2-selinux-files-etc". That link suggested **fixfiles relabel** to restore the security context of the files. I then googled **fixfiles relabel** to see how it's done and arrived here "https://www.theskillpedia.com/using-fixfiles-and-setting-mount-contexts-for-selinux/".

   I then changed back my modification to selinux config and did a sudo **fixfiles onboot**. Upon rebooting, the system did something like a lot of writing, then rebooted by itself again. To my surprise, It gave me my login prompt without a fuzz!

I think the problem is solved... unless you experts say otherwise. Please let me hear from you.

Thank you so much!

**Comment by steelwing** [comment]
Experiencing the exact same issue with inability to login following installation.

**Reply by mbaki** [reply]
I had the same issue, my password consisted of a $ and !, had to reinstall and assign a password without those special characters

**Reply by steelwing** [reply]
That worked, mbaki!  Thanks for the help!
Devs, this NEEDS to be part of the instructions and part of the onscreen prompts.  Like, really really REEEEEAAAALLY needs!  That, or you need to fix whatever in your install script prevents the use of $ and ! in the initial password.

**Comment by dougburks** [comment, marked as answer]
I've added a warning to the Installation documentation:
https://docs.securityonion.net/en/2.3/installation.html

I've also created an issue to fix this:
https://github.com/Security-Onion-Solutions/securityonion/issues/4249
